stock_move_actual_date/models/actual_date_mixin.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ActualDateMixin(models.AbstractModel):
+    _name = "actual.date.mixin"
+
+    actual_date = fields.Date(
+        tracking=True,
```

Comment:
```suggestion
        tracking=True,
        copy=False,
```
Otherwise, backorders keep the actual date.